plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts

```diff
@@ -196,9 +196,15 @@ export class ExpressInstrumentation extends InstrumentationBase<
         const route = (req[_LAYERS_STORE_PROPERTY] as string[])
           .filter(path => path !== '/' && path !== '/*')
           .join('');
-        const attributes: SpanAttributes = {
-          [SemanticAttributes.HTTP_ROUTE]: route.length > 0 ? route : '/',
-        };
+        const attributes: SpanAttributes = instrumentation._getSpanAttributes(
+          {
+            request: req,
+            route,
+          },
```

Comment:
`layerType` is missing on this object

Reply:
It's not passed either when setting the span name. Should I add it to both?

Reply:
It is [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/5f0d1e4e43412d6a6598e2b6703f0bead4f60d13/plugins/node/opentelemetry-instrumentation-express/src/instrumentation.ts#L239)

Reply:
Ah, indeed. I've changed the PR to properly include it (which brings back/explains the need for layerType in tests).